panda:
  init_x: 3.7275
  init_y: 0.47
  init_z: 0.072
  init_roll: 0.0
  init_pitch: 0.0
  init_yaw: 3.14159265359